pandas/_libs/tslibs/src/datetime/np_datetime.c

```diff
@@ -28,41 +28,6 @@ This file is derived from NumPy 1.7. See NUMPY_LICENSE.txt
 #include "np_datetime.h"
 
 
-const npy_datetimestruct _AS_MIN_DTS = {
```

Comment:
These were moved to the header file. The `extern` declaration puts the onus on the linker to resolve these symbols, which doesn't play nicely with setuptools. I am not aware of any downside to placing as `static const` in the header; upside is it plays nicer with our build system